src/main/java/org/reactome/addlinks/fileprocessors/COSMICFileProcessor.java

```diff
@@ -23,17 +25,23 @@ public COSMICFileProcessor(String processorName)
 	@Override
 	public Map<String, String> getIdMappingsFromFile()
 	{
-		Map<String,String> mappings = new HashMap<String, String>();
+		Map<String,String> mappings = new ConcurrentHashMap<>();
 		try
 		{
 			String unzippedFile = this.unzipFile(this.pathToFile);
 			AtomicInteger lineCount = new AtomicInteger(0);
-			try
+
+			try(Stream<String> lineStream = Files.lines(Paths.get(unzippedFile), StandardCharsets.ISO_8859_1))
```

Comment:
Just wondering, why the ISO_8859_1Character Set instead of UTF_8?

Reply:
Because it worked.